src/sprout/sprout_load.php

```diff
@@ -59,6 +59,20 @@
 Register::cronJob('daily', 'Sprout\\Controllers\\Admin\\FileAdminController', 'cronCleanupInvalid');
 Register::cronJob('daily', 'Sprout\\Controllers\\ContentSubscribeController', 'cronSendSubscriptions');
 Register::cronJob('daily', 'Sprout\\Controllers\\Admin\\ActionLogAdminController', 'cronCleanup');
+Register::cronJob('daily', 'Sprout\\Controllers\\RetentionCronController', 'cronRetention');
+
+// Purge exception log entries after 14 days
```

Comment:
These comments don't really add anything here, except maybe the fact that ISO 8601 period specs are a little opaque.

Of course we could switch away from ISO 8601 period specs and instead to `strtotime` relative time strings (e.g. `'14 days'`)